runtime/doc/news.txt

```diff
@@ -230,6 +230,10 @@ The following changes to existing APIs or features add new behavior.
   In addition, |nvim_buf_get_extmarks()| has gained an "overlap" option to
   return such ranges even if they started before the specified position.
 
+• |:range!| now pipe by default to |'shell'| on startup if a {filter} external
```

Comment:
Mentioning "on startup" in all of these docs is somewhat confusing and definitely unnecessary since it's an implementation detail. It's merely "the default".

```suggestion
• |:range!| pipes by default to |'shell'| if a {filter} external
```